### DISCUSSION TITLE
How to access globalstorage of extensionContext in extester?

### DISCUSSION THREAD

**Discussion by jitendra19** [redhat-developer/vscode-extension-tester#1666] [Q&A]
We are storing some information on ExtensionContext's global state. But while writing tests, not able to get context or global state.

In extension, while updating key in globalstate - 
`context.globalState.update('key', new Date());`

and while getting - 
`context.globalState.get('key');
`

We want to test whether particular "key" has some value in global state. But seems like there is no option to get extensionContext's globalstate in extester lib. 

Is there a way to access global state? TIA.

**Comment by djelinek** [comment]
Hello,
only option I can see from VS Code docs is a globalStorage which stores states for current instance. For test instance this files can be found under `TEST_RESOURCE` folder (e.g. under `$TMPDIR/test-resources` byt default)

e.g.
`/var/folders/b7/xf88srzs7ml8v05df2vx2vgh0000gn/T/test-resources/settings/User/globalStorage/storage.json`
